conda/environments/cuxfilter_dev_cuda10.0.yml

```diff
@@ -7,6 +7,7 @@ channels:
 - anaconda
 dependencies:
 - cupy>=6*
+- cuspatial>=0.15
```

Comment:
This file should be renamed and changed to CUDA 11.0 as we no longer support CUDA 10.0